tests/test_request.py

```diff
@@ -0,0 +1,20 @@
+from promoted_python_delivery_client.model.insertion import Insertion
+from promoted_python_delivery_client.model.request import Request
+from promoted_python_delivery_client.model.user_info import UserInfo
+
+
+def test_to_json_has_no_nones():
```

Comment:
I assume more unit tests in a different PR.

Reply:
There are a lot of unit tests in the Java SDK that I mostly plan to port but that will take at least two days and would make this PR even larger, yes.